actionpack/lib/action_dispatch/routing/inspector.rb

```diff
@@ -45,12 +45,22 @@ def action
       end
 
       def internal?
-        controller.to_s =~ %r{\Arails/(info|mailers|welcome)} || path =~ %r{\A#{Rails.application.config.assets.prefix}\z}
+        internal_controller? || internal_asset?
```

Comment:
The second part of this method has been dropped in 757a2bc3e3e52a5d9418656928db993db42b741b, probably best to just revert it to the current rails master version, since these changes don't seem related to `rails-api` and the erroneous `\a` => `\A` change below.